tests/test_boilerplate_removed.py

```diff
@@ -0,0 +1,64 @@
+"""
+This file checks that all the example boilerplate text has been removed.
+It can be deleted when all the contained tests pass
+"""
+import configparser
+from pathlib import Path
+
+ROOT = Path(__file__).parent.parent
+
+
+def skeleton_check(check: bool, text: str):
+    if ROOT.name == "python3-pip-skeleton" or str(ROOT) == "/project":
+        # In the skeleton module the check should fail
+        check = not check
+        text = f"Skeleton didn't raise: {text}"
+    if check:
+        raise AssertionError(text)
+
+
+def assert_not_contains_text(path: str, text: str, explanation: str):
+    full_path = ROOT / path
+    if full_path.exists():
+        contents = full_path.read_text().replace("\n", " ")
+        skeleton_check(text in contents, f"Please change ./{path} {explanation}")
+
+
+def assert_not_exists(path: str, explanation: str):
+    exists = (ROOT / path).exists()
+    skeleton_check(exists, f"Please delete ./{path} {explanation}")
```

Comment:
Can delete this when `test_explanations_written()` deleted

Reply:
done